WarOOP/Models/Warrior.cs

```diff
@@ -17,7 +17,22 @@ public Warrior()
         Attack = 5;
     }
 
-    private void GetDamageFrom(Warrior enemy) => CurrentHealth -= enemy.Attack;
+    protected virtual void GetDamageFrom(Warrior enemy)
+    {
+        if (IsAlive) {CurrentHealth -= enemy.Attack;}
```

Comment:
Adding curly braces also means that you need to format the code. Now it's not formatted